benchmarks/aggregate.py

```diff
@@ -56,9 +64,26 @@ def skip_model(args, model_name: str):
           re.search("|".join(args.exclude), model_name, re.I))
 
 
+def get_backend_name(dynamo: str, xla: str) -> str:
+  if dynamo == 'inductor':
+    return dynamo
```

Comment:
nitty nit: return 'inductor', so it's more explicit and you do not have to look at the line above.

Reply:
Done, thanks